$MirOS: ports/www/dillo/v2/patches/patch-dpi_cookies_c,v 1.3 2009/08/24 16:54:23 tg Exp $
--- dpi/cookies.c.orig	Tue Feb  2 22:15:50 2010
+++ dpi/cookies.c	Sun Feb 21 18:22:43 2010
@@ -238,13 +238,13 @@ static void Cookies_init()
    }
 
    /* Get a stream for the cookies file */
-   filename = dStrconcat(dGethomedir(), "/.dillo/cookies.txt", NULL);
+   filename = dStrconcat(dGethomedir(), "/.etc/dillo/cookies.txt", NULL);
    file_stream = Cookies_fopen(filename, "r+", cookies_txt_header_str);
 
    dFree(filename);
 
    if (!file_stream) {
-      MSG("ERROR: Can't open ~/.dillo/cookies.txt; disabling cookies\n");
+      MSG("ERROR: Can't open ~/.etc/dillo/cookies.txt; disabling cookies\n");
       return;
    }
 
@@ -1166,7 +1166,7 @@ static int Cookie_control_init(void)
    bool_t enabled = FALSE;
 
    /* Get a file pointer */
-   filename = dStrconcat(dGethomedir(), "/.dillo/cookiesrc", NULL);
+   filename = dStrconcat(dGethomedir(), "/.etc/dillo/cookiesrc", NULL);
    stream = Cookies_fopen(filename, "r", "DEFAULT DENY\n");
    dFree(filename);
 
